app/view_objects/results_view.rb

```diff
@@ -147,14 +147,22 @@ def courses
 
                    base_query
                      .page(query_parameters[:page] || 1)
-                     .per(10)
+                     .per(per_page)
                  end
   end
 
   def course_count
     courses.meta["count"]
   end
 
+  def per_page
```

Comment:
pointless method?
`ITEMS_PER_PAGE = 10`

Reply:
It's being used below (line 163) @defong...

Reply:
https://github.com/DFE-Digital/find-teacher-training/blob/d739a4e37e575ca1a49f7fe446fab664217ab409/app/view_objects/results_view.rb#L7-L11

Its a static value so pointless method in the sense that its a constant like the others?

@dankmitchell 

https://kirshatrov.com/2019/08/20/methods-vs-constants/

Reply:
this is now a constant but referenced in the method. it is a also now a private method